docs/README.md

```diff
@@ -483,6 +483,11 @@ ${execpi 300 gcalcli --conky calw 3}
 You may need to increase the `text_buffer_size` in your conkyrc file.  Users
 have reported that the default of 256 bytes is too small for busy calendars.
 
+Additionaly you need to set `--lineart=unicode` to output unicode-characters for box drawing. To avoid misaligned borders use a monospace font like 'DejaVu Sans Mono'. On Python2 it might be necessary to set the environment variable `PYTHONIOENCODING=utf8` if you are using characters beyond 'latin1'. For example:
```

Comment:
'latin1' characters will cause trouble, so this really ought o say 'beyond 'ascii'.